python/mqt/syrec/simulation_view/styled_item_delegates/simulation_run_overview_styled_item_delegate.py

```
@@ -0,0 +1,565 @@
+            text_color=QtCore.Qt.GlobalColor.gray,
+        )
+        painter.restore()
+
+    @staticmethod
+    def _draw_card_border_and_header(
+        painter: QtGui.QPainter,
+        option: QtWidgets.QStyleOptionViewItem,
+        simulation_run_number: int,
+        card_content_rect: QtCore.QRect,
+        available_header_width: int,
+        draw_rect_corners: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)